nut/DOCS.md

```diff
@@ -225,6 +225,21 @@ specific drivers, should not be changed for the majority of users.
 Allows setting the DEADTIME value in upsmon.conf to adjust the stale time for
 the monitor process, should not be changed for the majority of users.
 
+### Option: `upsmon_pollfreq`
+
+Allows setting the POLLFREQ value in upsmon.conf to adjust poll frequency for
+the monitor process. By default, upsmon uses 5 seconds. If this is flooding
```

Comment:
This should be rephrased, what does "flooding" mean?

Reply:
This is reused phraseology from the [official documentation](https://networkupstools.org/docs/man/upsmon.conf.html). But could be changed, of course :)